MASH-FRET/source/traces/processing/FRET/calcFRET.m

```diff
@@ -98,18 +98,41 @@
             I_eq{d}{c} = str_I;
             if ~isempty(strfind(str_I, sprintf('E_%i%i', d, c)))
                 
-                symvar(1) = sym('Ic');
-                symvar(2) = sym('I_0');
-                symvar(3) = sym(sprintf('E_%i%i', d, c));
-                evar_str = findEelsethan(str_I,sprintf('E_%i%i',d,c));
-                for i_e = 1:size(evar_str,2)
-                    symvar(3+i_e) = sym(evar_str{i_e});
+                % check for Matlab version to use sym variables
```

Comment:
Thanks. This came up with Bishnu Paudel from van Oijen group using Matlab version 2014a.